content/ru/cloud-native-tech.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Нативная облачная технология
+status: Completed
+category: Concept
+tags: ["fundamental", "", ""]
+---
+
+## Описание
+
+Нативные облачные технологии, также называемые нативным облачным стеком, — 
+это технологии, используемые для создания [нативных облачных приложений](/cloud-native-apps/).
+Эти технологии позволяют компаниям создавать и запускать масштабируемые приложения в современных и динамичных средах, 
+таких как публичные, приватные и гибридные облака, 
+максимально используя преимущества [облачных вычислений](/cloud-computing/).
+Они изначально разрабатываются для максимального использования возможностей облачных вычислений, и примером такого подхода являются контейнеры, сервисные сетки, микросервисы и неизменяемая (immutable) инфраструктура.
```

Comment:
Based on the discussion with our most relevant engineers, we would stick to following the original "service mesh" naming in Russian.
```suggestion
Они изначально разрабатываются для максимального использования возможностей облачных вычислений, и примером такого подхода являются контейнеры, сервис-меши, микросервисы и неизменяемая (immutable) инфраструктура.
```